winit-web/src/web_sys/pointer.rs

```diff
@@ -186,46 +207,48 @@ impl PointerHandler {
             Some(canvas_common.add_event("pointermove", move |event: PointerEvent| {
                 let pointer_id = event.pointer_id();
                 let device_id = mkdid(pointer_id);
-                let kind = event::pointer_type(&event, pointer_id);
+                let kind = event::pointer_kind(&event, pointer_id);
                 let primary = event.is_primary();
 
                 // chorded button event
-                if let Some(button) = event::mouse_button(&event) {
+                if let Some(button) = event::raw_button(&event) {
                     if prevent_default.get() {
                         // prevent text selection
                         event.prevent_default();
                         // but still focus element
                         let _ = canvas.focus();
                     }
 
-                    let state = if event::mouse_buttons(&event).contains(button.into()) {
+                    let state = if event::pointer_buttons(&event)
+                        .contains(ButtonsState::from_bits_retain(button))
+                    {
                         ElementState::Pressed
                     } else {
                         ElementState::Released
                     };
 
                     let button = match kind {
-                        PointerKind::Mouse => ButtonSource::Mouse(button),
+                        PointerKind::Mouse => ButtonSource::Mouse(event::mouse_button(button)),
                         PointerKind::Touch(finger_id) => {
-                            let button_id = mouse_button_to_id(button);
-
-                            if button_id != 1 {
-                                tracing::error!("unexpected touch button id: {button_id}");
+                            if button != 0 {
+                                tracing::error!("unexpected touch button id: {button}");
                             }
```

Comment:
Original had `!= 1` but [W3C](https://w3c.github.io/pointerevents/#the-button-property) says Touch Contact would be 0